packages/types/src/sdk/datasources.ts

```diff
@@ -216,4 +216,5 @@ export interface DatasourcePlus extends IntegrationBase {
     filter?: string[]
   ): Promise<Schema>
   getTableNames(): Promise<string[]>
+  getViews?(): Promise<{ name: string; definition: string }[]>
```

Comment:
As far as I can see, this **definition** property is not used. Assuming that is the case, I would refactor this to match the method above:

```ts
getViewNames?(): Promise<string[]>
```